### PR TITLE
[DQL-170] Enable a modified version of the 5 star linked data rating system

### PR DESCRIPTION
**Pull request by salsa-nathan**
This PR contains changes for the following Jira stories:

- [DQL2-10] 5 Star rating system - Frontend enhancements (Timebox)

@duttonw & @ThrawnCA for review.

### CODE REVIEW

File: ckanext/data_qld_theme/templates/qa/openness_stars.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{#
+Displays 5 starts of openness for a resource or dataset with explanation
```
Comment (ThrawnCA) [inline ckanext/data_qld_theme/templates/qa/openness_stars.html:2]: Misspelled "stars".

File: ckanext/data_qld_theme/templates/qa/openness_stars_brief.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{#
+Displays 5 starts of openness for a resource or dataset
```
Comment (ThrawnCA) [inline ckanext/data_qld_theme/templates/qa/openness_stars_brief.html:2]: Misspelled "stars".

Comment (MarkCalvert) [reply ckanext/data_qld_theme/templates/qa/openness_stars_brief.html]: Fixed spelling mistake.
FYI this spelling mistake was coped from original template https://github.com/ckan/ckanext-qa/blob/master/ckanext/qa/templates/qa/openness_stars_brief.html#L2

File: ckanext/data_qld_theme/templates/qa/openness_stars.html (hunk 1 above)
Comment (MarkCalvert) [reply ckanext/data_qld_theme/templates/qa/openness_stars.html]: Fixed spelling mistake.
FYI this spelling mistake was copied from original template https://github.com/ckan/ckanext-qa/blob/master/ckanext/qa/templates/qa/openness_stars.html#L2

**Comment by MarkCalvert** [comment]
@ThrawnCA It was designed this way to keep any dataqld specific changes out of the forked repos so that only generic changes are in the forked repo to easily contribute back. So any dataqld specific template overrides which are not being contributed back are in the theme. This will hopefully make QA forked repository to be easily maintained when pushing/pulling changes upstream.

**Pull request merged**